src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -175,6 +183,45 @@ def test_parallellooptrans_refuse_codeblock():
     assert ("Nodes of type 'CodeBlock' cannot be enclosed "
             "by a OMPParallelLoopTrans transformation" in str(err.value))
 
+
+# Tests for OMPSingleTrans
+def test_ompsingle():
+    ''' Generic tests for the OMPSingleTrans transformation class '''
+    trans = OMPSingleTrans()
+    assert trans.name == "OMPSingleTrans"
+    assert str(trans) == "Insert an OpenMP Single region"
+
+    assert not trans.omp_nowait
```

Comment:
Picky but I prefer to be explicit so test that `trans.omp_nowait is True` and similarly `is False` at line 196.

Reply:
Ok - I had those originally and pycodetest complained (or maybe it was pylint? I don't remember). I can change them back if you'd prefer though!

Reply:
Done - I think it was when I had `==` that they complained but `is X` works :)